package io.kyberorg.yalsee.exception;

import io.kyberorg.yalsee.ui.err.IdentNotFoundView;

/**
 * Exception, which not intended to be thrown, just {@link IdentNotFoundView} handles it.
 *
 * @since 2.7.4
 */
public class IdentNotFoundException extends YalseeException {
}
